src/containers/App.tsx

```diff
@@ -77,13 +77,13 @@ function App() {
           />
           <Route
             exact
-            path="/events/:eventId/ccps/:ccpId/patients/new/:barcodeValue?"
-            component={(props) => <PatientProfilePage mode="new" {...props} />}
+            path="/events/:eventId/ccps/:ccpId/patients/:patientId/edit"
+            component={(props) => <PatientProfilePage mode="edit" {...props} />}
           />
           <Route
             exact
-            path="/events/:eventId/ccps/:ccpId/patients/:patientId"
-            component={(props) => <PatientProfilePage mode="edit" {...props} />}
+            path="/events/:eventId/ccps/:ccpId/patients/new/:barcodeValue?"
+            component={(props) => <PatientProfilePage mode="new" {...props} />}
```

Comment:
we can nix these route changes now - aaron's PR https://github.com/uwblueprint/paramedics-react/pull/29 fixed these issues so this is no longer necessary